ferrilata-devilline.IntegrationTests/Scenarios/IntegrationTests.cs

```diff
@@ -1,4 +1,13 @@
+using System;

```

Comment:
Strings should be `string`s instead of var's!

Reply:
line 38 changed,